com.wso2telco.dep.hub.ussdapi/com.wso2telco.dep.hub.ussdapi.synapse/src/main/synapse-config/api/ussdAPI.xml

```diff
@@ -15,7 +15,7 @@
         </inSequence>
         <outSequence/>
         <faultSequence>
-	    	<sequence key="com.wso2telco.dep.common.response.unexpectedError.Sequence"/>
+	    	<sequence key="com.wso2telco.dep.hub.ussdapi.resource.mo.subscribe.notification.splitter.fault.Sequence"/>
```

Comment:
keep default fault sequence here, use splitter-fault-sequence only for splitting sequence